src/main/java/EntryPoint/dto/ResponseDTO.java

```diff
@@ -0,0 +1,31 @@
+package EntryPoint.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDateTime;
+
+
+@Data
+@AllArgsConstructor
+public class ResponseDTO {
+    private LocalDateTime timestamp;
```

Comment:
Can you use Instant class instead of LocalDateTime?

Reply:
resolved